src/main/java/world/bentobox/bentobox/database/objects/Table.java

```diff
@@ -8,9 +8,9 @@
 
 @Retention(RUNTIME)
 @Target(TYPE)
-/**
- * Annotation to explicitly name tables
- * @author tastybento
+/*
+  Annotation to explicitly name tables
+  @author tastybento
  * @since 1.14.0
  */
```

Comment:
We want Javadocs here, not comments:
```java
/**
  * Annotation to explicitly name tables.
  * @author tastybento
  * @since 1.14.0
  */
```

You might want to put the javadoc before the annotations. This might be why it told you that this was a "dangling" javadoc.

Reply:
Aha, that's what that's all about.